ktor-client/ktor-client-tests/common/test/io/ktor/client/tests/WebSocketTest.kt

```diff
@@ -51,19 +51,36 @@ class WebSocketTest : ClientLoader() {
     fun testCancel() = clientTests(listOf("Apache", "Android", "Js", "iOS")) {
```

Comment:
Nit: it feels a bit confusing to me that `clientTests(engines) { ... }` means `clientTests(skipEngines = engines ) { ... }`. I would add explicit parameter name, but feel free to ignore :)

Reply:
It wouldn't help I think. Maybe migration to `vararg` can help, but I'm not sure

Reply:
I don't know how varargs can help, not sure as well